staging/src/k8s.io/apiextensions-apiserver/go.mod

```diff
@@ -77,6 +77,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f // indirect
```

Comment:
is this dependency expected?
it's being pulled in different files

Reply:
These are modules importing k8s.io/apiserver/pkg/endpoints/metrics (which now imports k8s.io/apimachinery/pkg/util/proxy), so they're picking up the dependency transitively. If we want to stay with this approach to the fix, I should be able to avoid it by moving the status recorder declarations to a different package.